front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect, useState, useRef } from 'react'
+import Footer from './Footer'
+import Hamburger from './Hamburger'
+import './IndividualShow.css';
+import axios from 'axios';
+import { createMockUser, mockShowAPI, mockUserImage, mockShowImage } from './MockData'
+
+/*the component stores user's watched episode progress 
+by allowing to save the show's latest season and episode watched*/
+const ProgressData = ({ season, episode }) => {
+  const refSeason = useRef();
+  const refEpisode = useRef();
+  const saveProgressData = (() => {
+    //the function will be further developed in sprint 2
+    let progress = [];
+    var season = React.findDOMNode(this.refs.season).value;
+    var episode = React.findDOMNode(this.refs.episode).value;
+    progress.push(season);
+    progress.push(episode);
+  });
+  return (
+    <form onSubmit={(e) => saveProgressData()} >
+      <label for="season">Current Season: </label>
+      <input id="season" class="progress" value={season} ref={refSeason} />
+      <br />
+      <label for="episode">Current Episode:</label>
+      <input id="episode" class="progress" value={episode} ref={refEpisode} />
+      <br /><br />
+      <input type="submit" value="Save Progress" />
+    </form>
+  )
+}
+
+/*the component displays and stores user's chosen platform*/
+
+const PlatformData = () => {
+
+  const refNetflix = useRef();
+  const refPrime = useRef();
+  const refHulu = useRef();
+  const refApple = useRef();
+
+
+  const savePlatform = (() => {
+    //the function will be further developed in sprint 2
```

Comment:
This should make a request to Mockaroo to mock the platform being updated and not be empty

Reply:
Mockaroo (API) doesn't have a field for platform - only hard coded mock data; then does some one need to add platform to the schema?

Reply:
The platform field is in the user schema since we won't want to modify the database of shows with whatever platform only one user is watching a show on. You'll want to make a PATCH API call to Mockaroo to update the `shows` field of a user object. You'll have to look up a show by its id and update the platform field.

Reply:
ok

Reply:
Unfortunately, we need to be able to interact with the mock data in a way that simulates how our frontend would behave; as such, it should be done this sprint.

Reply:
For an example of a mockaroo patch request, I'd suggest checking the master branch's profile.js, line 64.